.travis.yml

```diff
@@ -13,7 +13,8 @@ script:
   - go test -v -race -coverprofile=coverage.txt -covermode=atomic -tags=integration
 
 after_success:
-  - bash <(curl -s https://codecov.io/bash)
+  - go get github.com/mattn/goveralls
+  - goveralls -coverprofile=coverage.txt
```

Comment:
🙏

Reply:
What's wrong with codecov 😄

Reply:
Pricing model. We've decided to switch to coveralls because of that :)